pkg/clusteragent/admission/mutate/autoinstrumentation/config.go

```diff
@@ -110,6 +114,12 @@ func NewConfig(datadogConfig config.Component) (*Config, error) {
 		return nil, err
 	}
 
+	libraryStorageMedium, err := parseLibraryStorageMedium(datadogConfig)
+	if err != nil {
+		return nil, err
+	}
+	setStorageMedium(libraryStorageMedium)
```

Comment:
Feel free to ignore for this:

---

The one thing I can think of to make this _kind of mutator-y_ if you were feeling the vibe for the future, is to have a `libraryStorageVolumeOption`. 

This has a bit more of a pattern for this to change in the future.

```go
type libraryStorageVolumeOption func(*corev1.Volume)

func libraryStorageOptionEmptyDirMedium(/*medium*/) libraryStorageOptionVolume {
   return func(v *corev1.Volume) { 
      // the stuff 
   }
}
```